modules/multilanguage-protocol/src/main/protobuf/multilanguage-protocol.proto

```diff
@@ -65,12 +66,21 @@ message GetStateReply {
   State state = 2;
 }
 
+message HealthCheckRequest {
+
+}
+
+message HealthCheckReply {
+  string status = 1;
+}
+
 service MultilanguageGatewayService {
   rpc ForwardCommand (ForwardCommandRequest) returns (ForwardCommandReply) {}
   rpc GetState (GetStateRequest) returns (GetStateReply) {}
 }
 
 service BusinessLogicService {
+  rpc HealthCheckCommand (HealthCheckRequest) returns (HealthCheckReply) {}
```

Comment:
@EldarKZ16 I think you can call this just HealthCheck instead of HealthCheckCommand

Reply:
I think we'd want to health check the sidecar so this might want to be on the `MultilanguageGatewayService`?

Reply:
@jeffboutotte we'll eventually add it to both .... the Surge sidecar ("multilanguage server") is going to ping the business app ... and the business app is going to ping the Surge sidecar ....